mongodbatlas/data_federation.go

```diff
@@ -33,6 +34,10 @@ type DataFederationService interface {
 	Create(context.Context, string, *DataFederationInstance) (*DataFederationInstance, *Response, error)
 	Update(context.Context, string, string, *DataFederationInstance, *DataFederationUpdateOptions) (*DataFederationInstance, *Response, error)
 	Delete(context.Context, string, string) (*Response, error)
+	ListQueryLimits(context.Context, string, string) ([]*DataFederationQueryLimit, *Response, error)
+	GetQueryLimit(context.Context, string, string, string) (*DataFederationQueryLimit, *Response, error)
+	ConfigureQueryLimit(context.Context, string, string, string, *DataFederationQueryLimit) (*DataFederationQueryLimit, *Response, error)
```

Comment:
Generally, we use Update for `Patch`
```suggestion
	UpdateQueryLimit(context.Context, string, string, string, *DataFederationQueryLimit) (*DataFederationQueryLimit, *Response, error)
```

Reply:
@andreaangiolillo The same PATCH API is used to create and update this entity. Should we still change this to Update? Could that be misleading?